slammin-newz/src/components/WeeklyNews.jsx

```diff
@@ -24,42 +24,42 @@ import {Container} from 'react-bootstrap'
 // console.log(API_KEY)
 
 export default class WeeklyNews extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // set world news list to null to start for page load
+      weeklynewslist: null,
+    };
+  }
 
-    constructor(props) {
-        super(props)
-        this.state = {
-            // set world news list to null to start for page load
-            weeklynewslist: null
-        }
-        this.findNews = this.findNews.bind(this)
```

Comment:
removed this line, rest is white space

Reply:
to better ignore the linter stuff,
![Screen Shot 2020-08-18 at 6 06 50 PM](https://user-images.githubusercontent.com/38012334/90570454-b316d100-e17d-11ea-9d6d-de83aa45030c.png)